### PR TITLE
Added develop mode install on README.rst

### LINKED ISSUE #482: Update install procedure
This project should use either one of the commands below to be installed:

    Devs: pip install -r requirements/dev.txt
    End-users with venv willing to use git pull to update Kytos: pip install -r requirements/run.txt

This way, we can all have the same versions of all dependencies and, thus, the same software behavior.

### PR DESCRIPTION
**Pull request by viniarck**
Fixes #482

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -52,6 +52,13 @@ procedure:
    $ cd python-openflow
    $ sudo python3.6 setup.py install
 
+Alternatively, if you are a developer and want to install in develop mode:
+
+.. code-block:: shell
+
+   $ cd python-openflow
+   $ pip3.6 install -e .[dev] -e git://github.com/kytos/sphinx-theme.git#egg=kytos-sphinx-theme
```
[the remark this answers is not in the document]
Comment (hdiogenes) [reply README.rst]: I think what Beraldo means is: as `kytos-sphinx-theme` is already included in `requirements/dev.txt`, is it really needed here?

My request would be to add quotes around `.[dev]`, so it won't break under `zsh`: `pip3.6 install -e ".[dev]"`

**Comment by viniarck** [comment]
As of now, just a `pip install -e ".[dev]"` doesn't install `kythos-sphinx-theme` (which is why I added an extra `-e` fetching from git):

```
❯ pip install -e ".[dev]"
Obtaining file:///tmp/python-openflow
Collecting coverage (from python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/a4/5c/b3eac6aa7eb4af83c8580d9c1578343910bdcc59e64e3565402f9a527f9c/coverage-4.5.2-cp36-cp36m-manylinux1_x86_64.whl
Collecting tox (from python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/d0/43/2160a300e0b77a929a980f36ac7427dcef8f4ddac7a8c21e5a8baedad828/tox-3.5.3-py2.py3-none-any.whl
Collecting pip-tools (from python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/f7/58/7a3c61ff7ea45cf0f13f3c58c5261c598a1923efa3327494f70c2d532cba/pip_tools-3.1.0-py2.py3-none-any.whl
Collecting yala (from python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/58/17/b4b770fc75aa44e3824175d45c4228cf4f52ac1a344608a3db0908ceb7c4/yala-1.7.0-py3-none-any.whl
Collecting six<2,>=1.0.0 (from tox->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Requirement already satisfied: setuptools>=30.0.0 in ./.direnv/python-3.6.6/lib/python3.6/site-packages (from tox->python-openflow==2018.2b2) (40.6.2)
Collecting py<2,>=1.4.17 (from tox->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/3e/c7/3da685ef117d42ac8d71af525208759742dd235f8094221fdaafcd3dba8f/py-1.7.0-py2.py3-none-any.whl
Collecting filelock<4,>=3.0.0 (from tox->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/d7/ca/3c74396a9ed8a4cfab5459800edeef9a1269591cb21f5a49bd71a49c5fa2/filelock-3.0.10-py3-none-any.whl
Collecting virtualenv>=1.11.2 (from tox->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/7c/17/9b7b6cddfd255388b58c61e25b091047f6814183e1d63741c8df8dcd65a2/virtualenv-16.1.0-py2.py3-none-any.whl
Collecting toml>=0.9.4 (from tox->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/a2/12/ced7105d2de62fa7c8fb5fce92cc4ce66b57c95fb875e9318dba7f8c5db0/toml-0.10.0-py2.py3-none-any.whl
Collecting pluggy<1,>=0.3.0 (from tox->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/1c/e7/017c262070af41fe251401cb0d0e1b7c38f656da634cd0c15604f1f30864/pluggy-0.8.0-py2.py3-none-any.whl
Collecting click>=6 (from pip-tools->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl
Collecting docopt (from yala->python-openflow==2018.2b2)
Collecting pylint (from yala->python-openflow==2018.2b2)
  Downloading https://files.pythonhosted.org/packages/a5/06/ecef826f319055e6b231716730d7f9047dd7524ffda224b521d989f085b6/pylint-2.2.2-py3-none-any.whl (750kB)
    100% |████████████████████████████████| 757kB 6.8MB/s
Collecting isort (from yala->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/1f/2c/22eee714d7199ae0464beda6ad5fedec8fee6a2f7ffd1e8f1840928fe318/isort-4.3.4-py3-none-any.whl
Collecting pycodestyle (from yala->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/e5/c6/ce130213489969aa58610042dff1d908c25c731c9575af6935c2dfad03aa/pycodestyle-2.4.0-py2.py3-none-any.whl
Collecting pydocstyle (from yala->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/6e/3f/9207e1af3fb23c71a2a17a5d82d4c68f4dd6b3429b58514c74b95f6e0784/pydocstyle-3.0.0-py3-none-any.whl
Collecting mccabe (from pylint->yala->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/87/89/479dc97e18549e21354893e4ee4ef36db1d237534982482c3681ee6e7b57/mccabe-0.6.1-py2.py3-none-any.whl
Collecting astroid>=2.0.0 (from pylint->yala->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/fc/53/8809bc008bad0300897281a7b320b286dc0e84e836396c0cff6279841e8a/astroid-2.1.0-py3-none-any.whl
Collecting snowballstemmer (from pydocstyle->yala->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/d4/6c/8a935e2c7b54a37714656d753e4187ee0631988184ed50c0cf6476858566/snowballstemmer-1.2.1-py2.py3-none-any.whl
Collecting typed-ast; python_version < "3.7" and implementation_name == "cpython" (from astroid>=2.0.0->pylint->yala->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/35/69/94dd92985792df90cf3b544d33315e60a95cc1c7ce0518f73b4cc11f106e/typed_ast-1.1.0-cp36-cp36m-manylinux1_x86_64.whl
Collecting wrapt (from astroid>=2.0.0->pylint->yala->python-openflow==2018.2b2)
Collecting lazy-object-proxy (from astroid>=2.0.0->pylint->yala->python-openflow==2018.2b2)
  Using cached https://files.pythonhosted.org/packages/65/1f/2043ec33066e779905ed7e6580384425fdc7dc2ac64d6931060c75b0c5a3/lazy_object_proxy-1.3.1-cp36-cp36m-manylinux1_x86_64.whl
Installing collected packages: coverage, six, py, filelock, virtualenv, toml, pluggy, tox, click, pip-tools, docopt, mccabe, typed-ast, wrapt, lazy-object-proxy, astroid, isort, pylint,
 pycodestyle, snowballstemmer, pydocstyle, yala, python-openflow
  Running setup.py develop for python-openflow
Successfully installed astroid-2.1.0 click-7.0 coverage-4.5.2 docopt-0.6.2 filelock-3.0.10 isort-4.3.4 lazy-object-proxy-1.3.1 mccabe-0.6.1 pip-tools-3.1.0 pluggy-0.8.0 py-1.7.0 pycodes
tyle-2.4.0 pydocstyle-3.0.0 pylint-2.2.2 python-openflow six-1.11.0 snowballstemmer-1.2.1 toml-0.10.0 tox-3.5.3 typed-ast-1.1.0 virtualenv-16.1.0 wrapt-1.10.11 yala-1.7.0

/tmp/python-openflow master
❯ pip freeze | grep -i kytos
-e git+https://github.com/kytos/python-openflow.git@0a04446b8b31d4ad333dc40d16a5c11063bb5766#egg=python_openflow

/tmp/python-openflow master
❯
```

File: README.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (beraldoleal) [reply README.rst]: yes @hdiogenes .

**Comment by hdiogenes** [comment]
We'll have to investigate why that's happening. For now, you can just use `pip install -r requirements/dev.txt`.

File: README.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (hdiogenes) [reply README.rst]: @viniarck: `pip3.6 install -r requirements/dev.txt` will fix it.

[the remark this answers is not in the document]
Comment (viniarck) [reply README.rst]: Feel free to push the changes as necessary @hdiogenes. Thanks.

**Comment by hdiogenes** [comment]
@viniarck looks OK to me, please just rebase your commits with the latest master so that we can merge this.

**Pull request merged**